src/utils/generators/createEventElem.tsx

```diff
@@ -0,0 +1,25 @@
+import { ExtendedEvent } from "../../types";
+import { getEventStyle } from "../getters/getEventStyle";
+
+export function createEventElem(extEvent: ExtendedEvent, extEvents: ExtendedEvent[], changeCurrentChosenEventId: Function) {
```

Comment:
This should probably be named `EventElement` and be inside `components` directory. This component just takes in event info as props and displays it, just as any other React component.